app/src/main/java/com/g/tragosapp/ui/MainFragment.kt

```diff
@@ -66,11 +66,7 @@ class MainFragment : Fragment(), MainAdapter.OnTragoClickListener {
                 }
                 is Resource.Failure -> {
                     binding.progressBar.visibility = View.GONE
-                    Toast.makeText(
-                        requireContext(),
-                        "Ocurrió un error al traer los datos ${result.exception}",
-                        Toast.LENGTH_SHORT
-                    ).show()
+                    context?.shortToast("Ocurrió un error al traer los datos ${result.exception}")
```

Comment:
Here context? could be replace with requireContext()

Reply:
For what? context? is good

Reply:
requireContext() already checks for nullability instead of calling context? you are guaranteed that it will not be null

Reply:
you wrong. context? not lead to NPE it null safe. requireContext() & context? have little differences. 
 requireContext() - if (context == null) {
            throw new IllegalStateException("Fragment " + this + " not attached to a context.");
        }

Reply:
context!! - lead to NPE